classes/log/td.php

```diff
@@ -4,9 +4,9 @@
 
 namespace Fluentd\Log;
 
-class Td extends \Fuel\Core\Log{
+class Td extends \Fuel\Core\Log {
 
-	public static function _init(){
+	public static function _init() {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Method name "_init" should not be prefixed with an underscore to indicate visibility
* Opening brace should be on a new line